software/base/src/test/java/brooklyn/entity/pool/AbstractServerPoolTest.java

```diff
@@ -72,7 +70,8 @@ public void setUp() throws Exception {
                 .configure(ServerPool.INITIAL_SIZE, getInitialPoolSize())
                 .configure(ServerPool.MEMBER_SPEC, EntitySpec.create(EmptySoftwareProcess.class)));
         poolApp.start(ImmutableList.of(location));
-        assertTrue(pool.getAttribute(Attributes.SERVICE_UP));
+        EntityTestUtils.assertAttributeEqualsEventually(pool, Attributes.SERVICE_UP, true);
```

Comment:
I thought that the entity was guaranteed to be running when `start` returned. Is this untrue?